action.yml

```diff
@@ -1,108 +1,19 @@
-name: Pros Build
-description: Automatically build pros projects
+name: 'PROS Build'
+description: 'Automatically build PROS templates'
+branding:
+  color: 'blue'
+  icon: 'box'
+  colorized: true
 
 inputs:
-  repository:
-    description: Target repository
+  multithreading:
     required: false
-    default: ${{ github.repository }}
-  library-path:
+    default: true
+    description: Whether to use multithreading when building the project (-j flag in make)
```

Comment:
Because that just enables or disables the -j flag in make. I heard previously (I think) from Discord that the -j flag sometimes can cause issues, so it's there if it's an issue